src/hooks/useDropdownOptions.test.ts

```diff
@@ -0,0 +1,74 @@
+import {renderHook} from '@testing-library/react-hooks';
+import {useDropdownOptions} from '.';
+import {Item} from './useDropdownOptions';
```

Comment:
I wasn't sure what this import was at first. 

What about:

```suggestion
import type {Item} from './useDropdownOptions';
```

"Back in the day" types were prefaced with the letter I to signify it was an interface or type. I know that's fallen out of favor, what's the general convention now for naming reusable Types?

Reply:
I don't think I've ever used `import type` before, had to look that one up. 

I'm not sure that there's an agreed upon convention beyond PascalCase now. I try to name them based on their use case, in this instance the arg is called `items`, so `Item` seemed fitting, but that is pretty general. `ItemModel` is a convention I've seen a couple times that's a bit more descriptive